src/components/Clock/Clock.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Clock.scss';
+import PropTypes from 'prop-types';
+
+export class Clock extends React.Component {
+  state = {
+    date: '',
```

Comment:
```suggestion
    date: new Date().toLocaleTimeString(),
```
Можно сразу дату поставить

Reply:
Good idea